source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -86,12 +86,12 @@ cy.get(...).find(...).should(...)
 The commands outside of the `.then()` will not run until all of the nested commands finish.
 
 {% note info %}
-By using callback functions we've created a closure. Closures enable us to keep references around to refer to work done in previous commands.
+By using callback functions we've created a {% url closure https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures %}. Closures enable us to keep references around to refer to work done in previous commands.
 {% endnote %}
 
 ## Debugging
 
-Using `.then()` functions is an excellent opportunity to use `debugger`. This can help you understand the order in which commands are run. This also enables you to inspect the objects that Cypress yields you in each command.
+Using `.then()` functions is an excellent opportunity to use {% url `debugger` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures %}. This can help you understand the order in which commands are run. This also enables you to inspect the objects that Cypress yields you in each command.
```

Comment:
Was this the wrong link?

Reply:
Darn, I thought I fixed it when I did a self review. I will fix it in any case... huh. Ta for spotting this.